routing/slash_in_parameter.rst

```diff
@@ -5,47 +5,47 @@ How to Allow a "/" Character in a Route Parameter
 =================================================
 
 Sometimes, you need to compose URLs with parameters that can contain a slash
-``/``. For example, take the classic ``/hello/{username}`` route. By default,
-``/hello/Fabien`` will match this route but not ``/hello/Fabien/Kris``. This
-is because Symfony uses this character as separator between route parts.
+``/``. For example, consider the ``/share/{token}`` route. If the ``token``
+value contains a ``/`` character this route won't match. This is because Symfony
+uses this character as separator between route parts.
 
-This guide covers how you can modify a route so that ``/hello/Fabien/Kris``
-matches the ``/hello/{username}`` route, where ``{username}`` equals ``Fabien/Kris``.
+This article explains how you can modify a route definition so that placeholders
+can contain the ``/`` character too.
 
 Configure the Route
 -------------------
 
-By default, the Symfony Routing component requires that the parameters
-match the following regex path: ``[^/]+``. This means that all characters
-are allowed except ``/``.
+By default, the Symfony Routing component requires that the parameters match
+the following regular expression: ``[^/]+``. This means that all characters are
+allowed except ``/``.
 
-You must explicitly allow ``/`` to be part of your parameter by specifying
-a more permissive regex path.
+You must explicitly allow ``/`` to be part of your placeholder by specifying
+a more permissive regular expression for it:
 
 .. configuration-block::
 
     .. code-block:: php-annotations
 
         use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 
-        class DemoController
+        class DefaultController
```

Comment:
Not sure about that. Imagine that this action is for sharing files/folders in Dropbox or sharing photo albums in Google ... and you need the token (hash) of the thing you are sharing. We probably wouldn't call that controller TokenController, but ShareController, FileController, PhotoAlbumController, etc.  In these cases, I prefer to keep the neutral DefaultController name to not overcomplicate the examples ... but I'd love to read more opinions about this. Thanks!